addons/survey/static/src/scss/survey_templates_results.scss

```diff
@@ -35,8 +38,8 @@
                 page-break-after: avoid;
             }
         }
-        .o_survey_results_question_wrapper:has(button.collapsed) {
-            display: none;
+        .o_survey_results_question_wrapper:has(div.collapsed) {
```

Comment:
As the CSS will be updated but the XML will probably not be, we should keep both rules in stable temporarily.